commit 9f421841f71b58811a6b98c4e7e655581eaf5d89
Author: Jussi Laako <jussi.laako@linux.intel.com>
Date:   Mon Aug 15 17:07:23 2016 +0300

    Reduced native

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d839f3f..72b2431 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -120,84 +120,9 @@ ELSE(X11_FOUND)
   MESSAGE(STATUS "Looking for XLib - not found")
 ENDIF(X11_FOUND)
 
-# DRM
-pkg_check_modules(DRM REQUIRED libdrm)
-IF(DRM_FOUND)
-  MESSAGE(STATUS "Looking for DRM - found at ${DRM_PREFIX} ${DRM_VERSION}")
-  INCLUDE_DIRECTORIES(${DRM_INCLUDE_DIRS})
-ELSE(DRM_FOUND)
-  MESSAGE(STATUS "Looking for DRM - not found")
-ENDIF(DRM_FOUND)
-
-include(CheckLibraryExists)
-# DRM Intel
-pkg_check_modules(DRM_INTEL libdrm_intel>=2.4.52)
-IF(DRM_INTEL_FOUND)
-  INCLUDE_DIRECTORIES(${DRM_INTEL_INCLUDE_DIRS})
-  MESSAGE(STATUS "Looking for DRM Intel - found at ${DRM_INTEL_PREFIX} ${DRM_INTEL_VERSION}")
-  CHECK_LIBRARY_EXISTS(drm_intel "drm_intel_bo_alloc_userptr" ${DRM_INTEL_LIBDIR} HAVE_DRM_INTEL_USERPTR)
-  IF(HAVE_DRM_INTEL_USERPTR)
-    MESSAGE(STATUS "Enable userptr support")
-  ELSE(HAVE_DRM_INTEL_USERPTR)
-    MESSAGE(STATUS "Disable userptr support")
-  ENDIF(HAVE_DRM_INTEL_USERPTR)
-  CHECK_LIBRARY_EXISTS(drm_intel "drm_intel_get_eu_total" ${DRM_INTEL_LIBDIR} HAVE_DRM_INTEL_EU_TOTAL)
-  IF(HAVE_DRM_INTEL_EU_TOTAL)
-    MESSAGE(STATUS "Enable EU total query support")
-  ELSE(HAVE_DRM_INTEL_EU_TOTAL)
-    MESSAGE(STATUS "Disable EU total query support")
-  ENDIF(HAVE_DRM_INTEL_EU_TOTAL)
-  CHECK_LIBRARY_EXISTS(drm_intel "drm_intel_get_subslice_total" ${DRM_INTEL_LIBDIR} HAVE_DRM_INTEL_SUBSLICE_TOTAL)
-  IF(HAVE_DRM_INTEL_SUBSLICE_TOTAL)
-    MESSAGE(STATUS "Enable subslice total query support")
-  ELSE(HAVE_DRM_INTEL_SUBSLICE_TOTAL)
-    MESSAGE(STATUS "Disable subslice total query support")
-  ENDIF(HAVE_DRM_INTEL_SUBSLICE_TOTAL)
-  CHECK_LIBRARY_EXISTS(drm_intel "drm_intel_get_pooled_eu" "" HAVE_DRM_INTEL_POOLED_EU)
-  IF(HAVE_DRM_INTEL_POOLED_EU)
-    MESSAGE(STATUS "Enable pooled eu query support")
-  ELSE(HAVE_DRM_INTEL_POOLED_EU)
-    MESSAGE(STATUS "Disable pooled eu query support")
-  ENDIF(HAVE_DRM_INTEL_POOLED_EU)
-  CHECK_LIBRARY_EXISTS(drm_intel "drm_intel_get_min_eu_in_pool" "" HAVE_DRM_INTEL_MIN_EU_IN_POOL)
-  IF(HAVE_DRM_INTEL_MIN_EU_IN_POOL)
-    MESSAGE(STATUS "Enable min eu in pool query support")
-  ELSE(HAVE_DRM_INTEL_MIN_EU_IN_POOL)
-    MESSAGE(STATUS "Disable min eu in pool query support")
-  ENDIF(HAVE_DRM_INTEL_MIN_EU_IN_POOL)
-ELSE(DRM_INTEL_FOUND)
-  MESSAGE(FATAL_ERROR "Looking for DRM Intel (>= 2.4.52) - not found")
-ENDIF(DRM_INTEL_FOUND)
-
-# CMRT
-pkg_check_modules(CMRT libcmrt)
-IF(CMRT_FOUND)
-INCLUDE_DIRECTORIES(${CMRT_INCLUDE_DIRS})
-ENDIF(CMRT_FOUND)
-
 # Threads
 Find_Package(Threads)
 
-IF(X11_FOUND)
-# Xext
-pkg_check_modules(XEXT REQUIRED xext)
-IF(XEXT_FOUND)
-  INCLUDE_DIRECTORIES(${XEXT_INCLUDE_DIRS})
-  MESSAGE(STATUS "Looking for Xext - found at ${XEXT_PREFIX}")
-ELSE(XEXT_FOUND)
-  MESSAGE(STATUS "Looking for Xext - not found")
-ENDIF(XEXT_FOUND)
-
-# Xfixes
-pkg_check_modules(XFIXES REQUIRED xfixes)
-IF(XFIXES_FOUND)
-  INCLUDE_DIRECTORIES(${XFIXES_INCLUDE_DIRS})
-  MESSAGE(STATUS "Looking for Xfixes - found at ${XFIXES_PREFIX}")
-ELSE(XFIXES_FOUND)
-  MESSAGE(STATUS "Looking for Xfixes - not found")
-ENDIF(XFIXES_FOUND)
-ENDIF(X11_FOUND)
-
 OPTION(ENABLE_GL_SHARING "cl_khr_gl_sharing" OFF)
 
 IF(ENABLE_GL_SHARING)
@@ -268,13 +193,6 @@ ENDIF(BUILD_EXAMPLES)
 
 ADD_SUBDIRECTORY(include)
 ADD_SUBDIRECTORY(backend)
-ADD_SUBDIRECTORY(src)
-ADD_SUBDIRECTORY(utests EXCLUDE_FROM_ALL)
-
-# compile benchmark only if standalone compiler is not provided
-IF (NOT (USE_STANDALONE_GBE_COMPILER STREQUAL "true"))
-  ADD_SUBDIRECTORY(benchmark EXCLUDE_FROM_ALL)
-ENDIF (NOT (USE_STANDALONE_GBE_COMPILER STREQUAL "true"))
 
 IF(BUILD_EXAMPLES)
 ADD_SUBDIRECTORY(examples)
